packages/@ourworldindata/grapher/src/stackedCharts/MarimekkoChart.test.ts

```diff
@@ -468,13 +468,13 @@ it("can filter years correctly", () => {
     ])
 
     // TODO: why is it ySlugs and xSlug here instead of yColumnSlugs and xColumnSlug? Unify when we have config migrations?
-    const manager = {
-        chartTypes: [GRAPHER_CHART_TYPES.Marimekko],
+    const manager: GrapherProgrammaticInterface = {
+        chartTypes: ["Marimekko"],
         table,
-        selection: table.availableEntityNames,
+        selectedEntityNames: table.availableEntityNames,
         ySlugs: "percentBelow2USD",
         xSlug: "population",
-        endTime: 2001,
+        maxTime: 2001,
```

Comment:
ooc, why these changes? 🙂 a change that was missed the first time around?

Reply:
`selection` and `endTime` are Grapher internals. Passing them in works, but it's generally better to use fields that are part of Grapher's supported interface